src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
I have to say that maybe the thing is that the message is not very self explanatory. The same happens in Spanish and French. Talking about `folder overlay` is maybe too abstract.  Maybe we should be more straight about what will happen. Something like: "Custom folder icons are going to be disabled/enabled"